ports/espressif/boards/espressif_esp32s3_devkitc_1_n32r8/mpconfigboard.mk

```diff
@@ -7,4 +7,4 @@ IDF_TARGET = esp32s3
 
 CIRCUITPY_ESP_FLASH_MODE=dout
 CIRCUITPY_ESP_FLASH_FREQ=80m
-CIRCUITPY_ESP_FLASH_SIZE=32MB
+CIRCUITPY_ESP_FLASH_SIZE=16MB
```

Comment:
Why change this? The board name says n32.

Reply:
this was not supposed to be here, it got added back by mistake. will revert.